documentation/objects/objects.md

```diff
@@ -53,31 +62,68 @@ If you already have binary segmentation images, skip this section.
+We also provide a view on the thresholded objects before size filtering.
+This layer is activated by checking the _Show intermediate results_ checkbox.
+In the example below, the result before size filter is shown in white.
+After application of the size filter, only the colored objects remain.
 
 <a href="figs/thresholding_before_size_filter.png" data-toggle="lightbox"><img src="figs/thresholding_before_size_filter.png" class="img-responsive" /></a>
 
-The **"Two Thresholds"** tab performs hysteresis thresholding with two thresholds: high and low. The high threshold is applied first and the resulting objects are filtered by size. For the remaining objects the segmentation is then relaxed to the level of low threshold. The two levels of thresholding allow to separate the criteria for detection and segmentation of objects and select only objects of very high probability while better preserving their shape. As for the single threshold case, we provide a view on the intermediate results after the application of the high threshold, the size filter and the low threshold. The image below shows the results of the high (detection) threshold in multiple colors overlayed with the results of the low (segmentation) threshold in white:
+
+### Hysteresis Thresholding
+
+By selecting **Hystersis** in the method drop-down thresholding is performed at two levels: _core, or high threshold_, and _final, or low threshold_.
+The high threshold is applied first and the resulting objects are filtered by size.
+These high probability areas act as seeds:
+For the remaining objects the segmentation is then relaxed to the level of low threshold starting from the seeds.
+The two levels of thresholding allow to separate the criteria for detection and segmentation of objects and select only objects of very high probability while better preserving their shape.
+As for the single threshold case, we provide a view on the intermediate results after the application of the high threshold, the size filter and the low threshold.
+The image below shows the results of the high (detection) threshold after size filter in pink (<span style="color:#ff0aff">&#9679;</span>) overlayed on the final results of the low (segmentation) threshold multiple colors:
 
 <a href="figs/thresholding_two_thresholds.png" data-toggle="lightbox"><img src="figs/thresholding_two_thresholds.png" class="img-responsive" /></a>
 
-The last parameter of this applet is the size filter, for which you can specify the minimum and maximum value. For both thresholding methods the end result is shown in the "Final output" layer.
+An additional setting is available with Hysteresis Thresholding:
+Two (or more) seeds (determined by the "core" threshold) might result in the same object when the segmentation is relaxed.
+With the _Don't merge objects_ checkbox, you can control this behavior:
+Checking it, will preserve one object per detection.
```

Comment:
How about pointing to the separated objects in this image which would get merged if the checkbox is not activate? The light pink and yellow one?